apps/gitlab/src/inngest/functions/users/delete-users.ts

```diff
@@ -0,0 +1,45 @@
+import { eq } from 'drizzle-orm';
+import { NonRetriableError } from 'inngest';
+import { db } from '@/database/client';
+import { organisationsTable } from '@/database/schema';
+import { inngest } from '@/inngest/client';
+import { deleteUser } from '@/connectors/users';
+import { decrypt } from '@/common/crypto';
+
+export const deleteSourceUsers = inngest.createFunction(
+  {
+    id: 'gitlab-delete-users',
+    concurrency: {
+      key: 'event.data.organisationId',
+      limit: 5,
```

Comment:
I just talk about it with @athielba we are going to use an env variable for this (with a default value of 5 it's fine) 

```suggestion
      limit: env.GITLAB_DELETE_USER_CONCURRENCY,
```